### ISSUE TITLE
Menu Help / About are worng on mac

### ISSUE THREAD

**Issue by GoogleCodeExporter** [Risehat/freemedforms#255]
```
Menu Help / About are worng on mac
```

Original issue reported on code.google.com by `eric.mae...@gmail.com` on 20 Dec 2012 at 5:52

**Comment by GoogleCodeExporter** [comment]
```
This issue was closed by revision 8cde8f9fddd6.
```

Original comment by `eric.mae...@gmail.com` on 20 Dec 2012 at 8:06
- Changed state: **Fixed**

**Comment by GoogleCodeExporter** [comment]
```
This issue was closed by revision 8607602ad0b1.
```

Original comment by `eric.mae...@gmail.com` on 20 Dec 2012 at 8:06
